### DISCUSSION TITLE
Rate limiting channel hopping

### DISCUSSION THREAD

**Discussion by tlfdev** [BearWare/TeamTalk5#2087] [Ideas]
We've had someone attacking our server via the TCP interface. He's written his spam script to jump between channels as fast as possible before disconnecting and reconnecting. This is spamming, annoying and frustrating. Can we have a way in server options to limit how fast or often someone can jump from channel to channel?

**Comment by bear101** [comment]
In user accounts there's an option to limit number of commands in a given interval.

Alternatively use the SpamBot: https://bearware.dk/?p=1433

**Comment by Ramon-Salazar** [comment]
That's so weird as hell, this type of attack is too old, someone used it already in a lot of servers and it seems you never noticed that, as a result it's reported many times and your server ain't the only server got attacked by someone who used the TCP Interface.

I've been informed by the spirit of the truth and wisdom that you ain't smart enough to combat these types of attack @sorressean.
